jpegenc/models/rgb2ycbcr/test_rgb2ycbcr.py

```diff
@@ -0,0 +1,148 @@
+#!/bin/python
+from myhdl import StopSimulation, block, Signal, ResetSignal, intbv, delay, instance, always_comb, always_seq
+from random import randrange
+from rgb2ycbcr import *
+from myhdl.conversion import verify
+
+def rgb_to_ycbcr(r, g, b):
+
+    ycbcr = [None for _ in range(3)]
+    ycbcr[0] = int(
+        round(Y_COEFF[0] *
```

Comment:
Do you know how the `*_COEFF` are calculated?  Can the coefficients be locally calculated rather than being defined as a constant (i.e. there should be some equation that defines the coefficients).